src/test/java/com/google/maps/GeolocationApiTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2014 Google Inc. All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.google.maps;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.google.maps.model.CellTower;
+import com.google.maps.model.GeolocationPostPayload;
+import com.google.maps.model.GeolocationResult;
+import com.google.maps.model.WifiAccessPoint;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.concurrent.TimeUnit;
+
+@Category(LargeTests.class)
+public class GeolocationApiTest extends AuthenticatedTest {
```

Comment:
As the API is marked as not supporting client IDs for authentication (in `ApiConfig.supportsClientId(..)` above), you will need to make this class extend from `KeyOnlyAuthenticatedTest`, otherwise it will attempt to run these tests using a client ID, where provided (i.e. in our automated tests).